templates/experiments/single_experiment_home.html

```diff
@@ -260,9 +260,7 @@ <h3 class="font-bold text-lg">Parent Experiments:
     <input type="radio" name="tab_group" role="tab" class="tab" aria-label="Routes" id="tab-routes"/>
     <div role="tabpanel" class="tab-content" id="content-routes">
       <div class="app-card">
-        {% if experiment.assistant %}
-          Assistants cannot be used as a router bot
-        {% elif can_make_child_routes %}
+        {% if can_make_child_routes %}
```

Comment:
reverted this for another PR. There might be assistant bots that are already routing bots. We first need people to fix that

Reply:
Is it easy to get a list of experiments & team spaces where we have assistants as routers? I would guess it's just the dimagi team and we could make the change pretty quickly